circle.yml

```diff
@@ -16,6 +16,7 @@ workflows:
       - unit-tests:
```

Comment:
I thought that was your concern 😅

Reply:
My concern was not to change the makefile commands to jacoco because we run them locally as well. On CI, we can run only the jacoco ones :upside_down_face:

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/3663#discussion_r508392501 Ohh you meant only exposing it in the `Makefile` to use it locally 👍